server/route.go

```diff
@@ -60,23 +62,25 @@ func buildRoutes(app *applicationHandler) http.Handler {
 		v1Router.Route("/v1", func(r chi.Router) {
 			r.Use(middleware.AllowContentType("application/json"))
 			r.Use(jsonResponse)
+			r.Use(requirePermission())
```

Comment:
Here we want to do a few things as well. 

- change `requireGroupAccess` to `requireGroup`
- move it to this top-level & add `groupID` to every non-GET request route. 
- Apply same to uiRouter.

Reply:
1 done, but why move it to top level?
we only need it when we're trying to access groups?

Reply:
@Subomi